stdpopsim/drosophila_melanogaster.py

```diff
@@ -195,18 +197,20 @@ def __init__(self):
         t_AE = 158000  # generations
         t_E1 = t_AE - 3400
 
-        metadata_afr = {
-            "name": "AFR_dmel",
-            "description": "African D. melanogaster population"
-        }
-        metadata_eu = {
-            "name": "EU_dmel",
-            "description": "European D. melanogaster population"
-        }
+        # metadata_afr = {
+        #     "name": "AFR_dmel",
+        #     "description": "African D. melanogaster population"
+        # }
+        # metadata_eu = {
+        # "name": "EU_dmel",
+        # "description": "European D. melanogaster population"
+        # }
 
```

Comment:
I'm not seeing any changes in here other than removing the metadata - is the idea to inherit from one of the constant models?

Reply:
needed to remove the meta-data. Not sure why but it was throwing errors.